pkg/auth/serviceaccount.go

```diff
@@ -66,6 +68,9 @@ func GetOIDCServiceAccountForResource(gvk schema.GroupVersionKind, objectMeta me
 			Annotations: map[string]string{
 				"description": fmt.Sprintf("Service Account for OIDC Authentication for %s %q", gvk.GroupKind().Kind, objectMeta.Name),
 			},
+			Labels: map[string]string{
+				sources.OIDCLabelKey: "",
```

Comment:
I am OK with using the same selector for the OIDC related resources (e.g. use it for the OIDC created role & rolebinding of the apiserversource), but could we then:
* move it to a more general package as of `source` (e.g. `auth`?!?)
* Rename the `OIDCTokenRoleLabelSelector` constant then to something more generic?